example/20-componentconfig-gardenlet.yaml

```diff
@@ -118,7 +118,6 @@ featureGates:
   HVPAForShootedSeed: true
   ManagedIstio: true
   APIServerSNI: true
-  ReversedVPN: true
```

Comment:
https://github.com/gardener/gardener/blob/5a69fa603d12ed6a31424bb7e4adbd25b5c73a08/docs/development/local_setup.md#L263 has to be adapted.

Reply:
I think you just need to adapt this text now that there is no `ReversedVPN` feature gate or annotation anymore. Probably you can drop this line entirely now.
Generally, I assume this setup will be dropped soon in favor of the other local setup variants, so let's not invest too much into it now.